sensu/Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN \
   && apt-get upgrade -y \
   && apt-get install -y sensu
 
+ADD docker/confd /etc/confd
+
 ADD  docker/my_init.d /etc/my_init.d
 RUN  chmod a+x /etc/my_init.d/*
 
```

Comment:
can you add line (rename it):
ENV SENSU_DEFAULT_LOG_LEVEL=warn

this will let you simplify the template